README.md

```diff
@@ -1,64 +1,78 @@
-<div align="center">
+<p align="center">
+  <picture>
+    <source
+      media="(prefers-color-scheme: dark)"
+      srcset="assets/wordmark/wordmark+slogan-light.svg"
```

Comment:
You should swap image sources for these two. Currently it's black in dark theme and white in light theme.